test/Dialect/Torch/maximize-value-semantics.mlir

```diff
@@ -261,3 +261,88 @@ func.func @viewlike$two_inputs_two_copies(%arg0: !torch.vtensor, %arg1: !torch.v
   %3 = torch.copy.to_vtensor %2 : !torch.vtensor
   return %3 : !torch.vtensor
 }
+
+// CHECK-LABEL:   func.func @multiple_aliases_independent(
```

Comment:
There are several edge cases in the code that would benefit from lit tests. For example, overlapping aliases because of `dim`, overlapping because of `start-end`, non-overlapping when one `end = -1`, use of `static_info_cast`.